workflows/funcscan.nf

```diff
@@ -3,6 +3,7 @@
     PRINT PARAMS SUMMARY
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
+include { validateParameters; paramsHelp; paramsSummaryLog; paramsSummaryMap; fromSamplesheet } from 'plugin/nf-validation'
 
 include { paramsSummaryLog; paramsSummaryMap } from 'plugin/nf-validation'
 
```

Comment:
include statements are duplicated here, I think it's enough with including `paramsSummaryLog`, `paramsSummaryMap` and `fromSamplesheet`

Reply:
not the help?

Reply:
`paramsHelp` and `validateParameters` used in `main.nf` since the template update